package exception.handling;

public class TestLogin {
	
	public static void main(String[] args) throws LoginException {
	//	try {
			authenticate("vijay");
		}
		//catch(LoginException e) {
			//System.out.println("Invalid user");
		
	

	private static void authenticate(String name) throws LoginException {
		if (!name.equals("admin"));
		
		LoginException e = new LoginException ();
		//try {
		throw e ;
		} 
		//catch (LoginException e1) {
			//System.out.println(e1.getMessage());
		
			
		}
		
	
	

